.github/workflows/build-deploy-docs.yml

```
@@ -29,12 +29,10 @@ jobs:
           file-path: './README.md'
           config-file: '.github/files/markdown.links.config.json'
 
-      - name: Setup mdBook
-        uses: peaceiris/actions-mdbook@v2
-        with:
-          mdbook-version: '0.4.51'
+      - uses: taiki-e/install-action@v2
+        with: { tool: mdbook }
```

As discussed, better be explicite

```suggestion
        with: { tool: mdbook,cargo-binstall }
```